src/olympia/addons/tests/test_views.py

```diff
@@ -251,6 +252,17 @@ def test_get_not_listed(self):
         assert data['is_disabled_by_developer'] is False
         assert data['is_disabled_by_mozilla'] is False
 
+        AddonRegionalRestrictions.objects.filter(
+            addon=self.addon).update(excluded_regions=['AB', 'CD', 'FR'])
+        # Regional restrictions should be processed after other permission
+        # handling, so something that would return a 401/403/404 without
+        # region restrictions would still do that.
+        response = self.client.get(self.url)
```

Comment:
Just a drive-by comment, but where is one of `['AB', 'CD', 'FR']` set as `X-Country-Code` for the request? I.e., what reason do we have to believe the test will care that these have been added to `excluded_regions`?

Reply:
It's a good point, I manually passed the header for `test_addon_regional_restrictions` below but forgot it there (and in a couple other places). I'll fix that.

Reply:
7226d9cd3e49743f4c28b8286a7f8e4ca6aff569